python-libraries/nanover-core/tests/test_autoconnect.py

```diff
@@ -46,34 +46,23 @@ def discoverable_imd_server():
         yield app_server
 
 
-@pytest.fixture
-def discoverable_imd_server():
-    """
-    Returns a discoverable iMD server discoverable on the free port.
-    """
-    DISCOVERY_PORT = 39421
```

Comment:
It is curious that this is not the default ESSD port. Maybe we did that to avoid interference with other servers on the network?

Reply:
I was also unsure about this, but it does looks like ESSD has no behavior for picking any available port (which makes sense, it's supposed to run on a default port so you can find it), so it makes sense to pick specific non-overlapping ports for testing them in parallel. Maybe there's a more explicit way to pick sequential port numbers in a fixture?

Reply:
Sequential ports would be ideal, but the non-default port is likely enough. (Would be worth documenting the reasoning, though)

Reply:
We have to use explicit non-default ports at least, because broadcast doesn't have the means to find an unused one. Are these changes sufficent?